api/src/wfl/module/covid.clj

```diff
@@ -110,82 +112,6 @@
   "Use `tx` to load the workload sink with `sink_type`."
   (fn [tx workload] (:sink_type workload)))
 
-(defn ^:private add-continuous-workload-record
```

Comment:
Can you move these functions back up here please - there're meant to be calling interface/mulit-methods and not concrete implementations.
The workload should not know which source/executor/sink implementations are being used.

Reply:
Moved back.